website.Rmd

```diff
@@ -2,17 +2,396 @@
+Everything beyond this point is a "nice to have".
+
+Overall, we recommend `vignette("pkgdown", package = "pkgdown")` as a good place to start, if you think you want to go beyond the basic defaults.
+
+In the sections below, we highlight a few areas that are connected to other topics in the book or customizations that are particularly rewarding.
+
 ## Logo
 
-Website is also a great way to show off your package logo.
+It's fun to have a package logo!
+In the R community, we have a strong tradition of hex stickers, so it can be nice to join in with a hex logo of your own.
+Keen R user Amelia McNamara [made herself a dress](https://www.amelia.mn/blog/misc/2019/08/17/Tidy-Dress.html) out of custom hex logo fabric and useR!
+2018 featured a [spectacular hex photo wall](https://www.mitchelloharawild.com/blog/hexwall/).
+
+Here are some resources to guide your logo efforts:
+
+-   The convention is to orient the logo with a vertex at the top and bottom, with flat vertical sides.
+
+-   If you think you might print stickers, make sure to comply with the *de facto* standard for sticker size.
+    [hexb.in](http://hexb.in/sticker.html) is a reliable source for the dimensions and also provides a list of potential vendors for printed stickers.
+
+    ```{r}
+    #| label: fig-hex-sticker-spec
+    #| echo: false
+    #| out.width: 35%
+    #| fig.cap: Standard dimensions of a hex sticker.
+    # https://github.com/terinjokes/StickersStandard/blob/master/assets/hex-image.png
+    knitr::include_graphics("diagrams/hex-image.png")
+    ```
+
+-   The [hexSticker package](https://cran.r-project.org/package=hexSticker) helps you make your logo from within the comfort of R.
+
+Once you have your logo, the `usethis::use_logo()` function places an appropriately scaled copy of the image file at `man/figures/logo.png` and also provides a copy-paste-able markdown snippet to include your logo in your `README`.
+pkgdown will also discover a logo placed in the standard location and incorporate it into your site.
+
+## Reference index
+
+pkgdown creates a function reference in `reference/` that includes one page for each `.Rd` help topic in `man/`.
+This is one of the first pages you should admire in your new site.
+As you look around, there are a few things to contemplate, which we review below.
+
+pkgdown executes all your examples (@sec-man-examples) and inserts the rendered results.
+We find this is a fantastic improvement over just showing the source code.
+This view of your examples can be eye-opening and often you'll notice things you want to add, omit, or change.
+If you're not satisfied with how your examples appear, this is a good time to review techniques for including code that is expected to error (@sec-man-examples-errors) or that can only be executed under certain conditions (@sec-man-examples-dependencies-conditional-execution).
+
+These help topics will be linked to from many locations within and, potentially, beyond your pkgdown site.
```

Comment:
Subsections for linking, index organisation, etc?

Reply:
added